app/controllers/games_controller.rb

```diff
@@ -20,5 +20,18 @@ def create
       render text: "Invalid Request", status: :unprocessable_entity
     end
   end
+
+  def update
+    @game = Game.find(params[:id])
+    if current_user.id == params[:jedi_user_id].to_i
+      @game.update(:sith_user_id => current_user.id)
```

Comment:
This is the old hash rocket syntax, which is fine by me, but worth pointing out.